cookietemple/create/templates/cli/cli_python/{{ cookiecutter.project_slug_no_hyphen }}/README.rst

```diff
@@ -1,47 +1,75 @@
-{% set is_open_source = cookiecutter.license != 'Not open source' -%}
-{% for _ in cookiecutter.project_name %}={% endfor %}
 {{ cookiecutter.project_name }}
-{% for _ in cookiecutter.project_name %}={% endfor %}
+===========================
 
-.. image:: https://github.com/{{ cookiecutter.github_username }}/{{ cookiecutter.project_slug }}/workflows/Build%20{{ cookiecutter.project_slug }}%20Package/badge.svg
-        :target: https://github.com/{{ cookiecutter.github_username }}/{{ cookiecutter.project_slug }}/workflows/Build%20{{ cookiecutter.project_slug }}%20Package/badge.svg
-        :alt: Github Workflow Build {{ cookiecutter.project_name }} Status
+|PyPI| |Python Version| |License| |Read the Docs| |Build| |Tests| |Codecov| |pre-commit| |Black|
 
-.. image:: https://github.com/{{ cookiecutter.github_username }}/{{ cookiecutter.project_slug }}/workflows/Run%20{{ cookiecutter.project_slug }}%20Tox%20Test%20Suite/badge.svg
-        :target: https://github.com/{{ cookiecutter.github_username }}/{{ cookiecutter.project_slug }}/workflows/Run%20{{ cookiecutter.project_slug }}%20Tox%20Test%20Suite/badge.svg
-        :alt: Github Workflow Tests Status
+.. |PyPI| image:: https://img.shields.io/pypi/v/{{ cookiecutter.project_slug }}.svg
+   :target: https://pypi.org/project/{{ cookiecutter.project_slug }}/
+   :alt: PyPI
+.. |Python Version| image:: https://img.shields.io/pypi/pyversions/{{cookiecutter.project_slug }}
+   :target: https://pypi.org/project/{{ cookiecutter.project_slug }}
+   :alt: Python Version
+.. |License| image:: https://img.shields.io/github/license/{{ cookiecutter.github_username }}/{{ cookiecutter.project_slug }}
+   :target: https://opensource.org/licenses/{{ cookiecutter.license }}
+   :alt: License
+.. |Read the Docs| image:: https://img.shields.io/readthedocs/{{ cookiecutter.project_slug }}/latest.svg?label=Read%20the%20Docs
+   :target: https://{{cookiecutter.project_name}}.readthedocs.io/
```

Comment:
why the `slug` in line 15 and `name` in 16?

Reply:
Mistake. Will fix.